geomesa-features/geomesa-feature-kryo/src/test/scala/org/locationtech/geomesa/features/kryo/json/JsonPathFilterFunctionTest.scala

```diff
@@ -219,5 +222,19 @@ class JsonPathFilterFunctionTest extends Specification {
       ECQL.toFilter("jsonPath('$.json[''bar(bar)''].[''boo(boo)'']') = 'hiss'").evaluate(sf) must beTrue
       ECQL.toFilter("jsonPath('$.json[''bar(bar)''][''boo(boo)'']') = 'hiss'").evaluate(sf) must beTrue
     }
+
+    "work with 2 parameters" in {
+      ECQL.toFilter("""jsonPath("json", 'foo') = 'bar'""").evaluate(sf) must beTrue
+      ECQL.toFilter("""jsonPath("json", 'bar.boo') = 'hiss'""").evaluate(sf) must beTrue
+    }
+
+    "extract property descriptors with 2 parameters" in {
```

Comment:
Just to ask, do the sub attribute/bracket combinations above work with the two parameter approach?

Reply:
yeah, it will just take the first attribute as a property name and then append the second attribute as a string